packages/nns/src/ledger.canister.ts

```diff
@@ -123,6 +128,29 @@ export class LedgerCanister {
     return response.Ok;
   };
 
+  /**
+   * Transfer ICP from the caller to the destination `Account`.
+   * Returns the index of the block containing the tx if it was successful.
+   *
+   * @throws {@link TransferError}
+   */
+  public icrc1Transfer = async (
+    request: Icrc1TransferRequest,
```

Comment:
nth: destruct `{fee, ...rest}`

Reply:
I'm trying to stay consistent with the existing `transfer` method above.